modules/modal/react/README.md

```diff
@@ -117,6 +117,18 @@ Default: `PopupPadding.l`
 
 Default: `true`
 
+### `shouldUsePortal: boolean`
```

Comment:
Since we've decided to put this in 4.0, can we remove this prop entirely and just make this the default?

Are there any cases where we wouldn't want someone to use a portaled modal?